lib/scripts/create_blank_subcollections.rb

```diff
@@ -0,0 +1,62 @@
+# Script for creating nested collections, for institutional collections.
+# Creates "x" 'template' collections, adding another "y" template
+# collections to each.
+# To use, pass in a collection id, a number for template collections amount,
+# and letter for child template collections amount. The 'letter' parameter is
+# used in the form of a range, starting from 'a'. i.e. if 'letter' is 'c',
+# range is a..c => a,b,c
+# Example to create 10 template collections with 3 collections for each:
+# current_sufia/ $ bundle exec rails runner lib/scripts/create_template_collections.rb "abc-123" 10 "c"
```

Comment:
Interesting choice to use a letter as input...  puts more work on the user though. And obviously if they forget and use a number, no subcollection will be created because numbers are before lowercase letters. Since you would be running the script it's fine.

Reply:
Yea, this is my attempt to shorten the title for child collections. If you have a better idea, I am open to it. We can open an issue for it. 
Also, 
```suggestion
# current_sufia/ $ bundle exec rails runner lib/scripts/create_blank_subcollections.rb "abc-123" 10 "c"
```